src/main/java/uk/gov/hmcts/reform/sscs/validators/SscsCaseValidator.java

```diff
@@ -250,7 +259,13 @@ private void checkMrnDetails(Appeal appeal, Map<String, Object> ocrCaseData, boo
 
         if (dwpIssuingOffice != null && appeal.getBenefitType() != null && appeal.getBenefitType().getCode() != null) {
 
-            Optional<OfficeMapping> officeMapping = dwpAddressLookupService.getDwpMappingByOffice(appeal.getBenefitType().getCode(), dwpIssuingOffice);
+            Optional<OfficeMapping> officeMapping = Optional.empty();
+            //TODO: remove when ucOfficeFeatureActive fully enabled.
+            if (!ucOfficeFeatureActive && Benefit.UC.getShortName().equals(appeal.getBenefitType().getCode())) {
```

Comment:
I don't think you need to make this change to the Validator - the change you have done in the Transformer should be sufficient.

Reply:
It is needed as commons behaviour has changed. Now UC having more than one office it is not getting a default office for validation.

```
public Optional<OfficeMapping> getDwpMappingByOffice(String benefitType, String dwpIssuingOffice) {
        log.info("looking up officeAddress for benefitType {} and dwpIssuingOffice {}", benefitType, dwpIssuingOffice);

        if (equalsIgnoreCase(dwpIssuingOffice, TEST_HMCTS_ADDRESS)) {
            return of(dwpMappings.getTestHmctsAddress());
        }
        OfficeMapping[] dwpOfficeMappings = getDwpOfficeMappings(benefitType);
        if (dwpOfficeMappings.length == 1) {
            return of(dwpOfficeMappings[0]);
        }
        String dwpIssuingOfficeSearch = isPipBenefit(benefitType)
                ? stripDwpIssuingOfficeForPip(dwpIssuingOffice) : dwpIssuingOffice;
        return getOfficeMappingByDwpIssuingOffice(dwpIssuingOfficeSearch, dwpOfficeMappings);
    }
```

I'd rather removing  

```
if (dwpOfficeMappings.length == 1) {
            return of(dwpOfficeMappings[0]);
        }
```

but that would be another story I guess.